frontend/src/component/personalDashboard/createChartOptions.ts

```diff
@@ -82,6 +82,11 @@ export const createBarChartOptions = (
     return {
         plugins: {
             legend: plugins?.legend,
+            // required to avoid the highlight plugin highlighting empty annotation
+            annotation: {
```

Comment:
even though we don't use annotations on this chart it is used in the network data usage. ChartJs has global registrations so this page can be affected.